packages/api-reference/src/components/Content/ReferenceEndpoint/RequestBody.vue

```diff
@@ -1,21 +1,39 @@
 <script setup lang="ts">
+import { ref } from 'vue'
+
 import type { RequestBody } from '../../../types'
 import Schema from '../Schema.vue'
 
-defineProps<{ requestBody?: RequestBody }>()
+const prop = defineProps<{ requestBody?: RequestBody }>()
+const selectedContentType = ref(
+  prop?.requestBody?.content
+    ? Object.keys(prop?.requestBody?.content)[0]
```

Comment:
do we need to do an empty check here incase `prop.requestBody.content = {}`? since we check `[0]` would that fail if it exists with nothing?